samples/react/README.md

```diff
@@ -0,0 +1,85 @@
+This project integrates monaco-kusto in a create-react-app application.
+
+The following changes were made to a default create-react-app application:
+
+1. Used script tags to import Kusto language service (this piece of code is C# code transpiled to javascript. it has several issues that prevent it from being imported as a regular import statement).
+
+2. Added a prepare task to package.json that copies over monaco and monaco-kusto to the public folder (this is done because we're using UMD to require monaco and monaco-kusto in runtime. otherwise, we would have to eject create-react-app and play with webpack config as explained [here](https://github.com/microsoft/monaco-editor/blob/master/docs/integrate-esm.md) )
```

Comment:
> Added a prepare task to package.json

I don't see a prepare task.

> Added a ... task ... that copies over monaco and monaco-kusto to the public folder

Looking at the public folder, I can see monaco-editor. Is it holding both monaco and monaco-kusto? Maybe it worth being more specific about what is being copied.

> this is done because we're using UMD to require monaco and monaco-kusto in runtime

I can see that monaco-kusto is using https://www.npmjs.com/package/@monaco-editor/react. 
Reading their ReadMe and [Examples](https://monaco-react.surenatoyan.com/), they also support downloading monaco from a CDN. For this sample, it might be simpler to let @monaco-editor/react to download monaco from the CDN.